index.html

```diff
@@ -0,0 +1,205 @@
+                <span class="texttype2">Back End Dev</span>
+                <img src="images/dot.png" alt="dot">
+                <span class="texttype2">2015</span>
+            </div>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="wbtn">
+                <ul class="wbtn1">
+                    <li class="wbtn3">HTML</li>
+                    <li class="wbtn3">CSS</li>                    
+                    <li class="wbtn3">JavaScript</li>
+                </ul>              
+            </div>                   
+            <button class="btn">See project</button>
+          </div>                       
+        </div>
+        <div class="hero">
+          <div class="col1">
+            <img class="img-hero" src="images/p3.png" alt="Project 3">
+        </div>
+        <div class="col2">
+            <h2 class="title2">Tonic</h2>
+            <div class="type">
+                <span class="texttype">CANOPY</span>
+                <img src="images/dot.png" alt="dot">
+                <span class="texttype2">Back End Dev</span>
+                <img src="images/dot.png" alt="dot">
+                <span class="texttype2">2015</span>
+            </div>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="wbtn">
+                <ul class="wbtn1">
+                    <li class="wbtn3">HTML</li>
+                    <li class="wbtn3">CSS</li>                    
+                    <li class="wbtn3">JavaScript</li>
+                </ul>              
+            </div>                  
+            <button class="btn">See project</button>   
+          </div>                     
+        </div>
+        <div class="hero2">
+          <div class="col1">
+            <img class="img-hero" src="images/p4.png" alt="Project 4">
+          </div>
+          <div class="col2">
+            <h2 class="title2">Multi-Post Stories</h2>
+            <div class="type">
+                <span class="texttype">CANOPY</span>
+                <img src="images/dot.png" alt="dot">
+                <span class="texttype2">Back End Dev</span>
+                <img src="images/dot.png" alt="dot">
+                <span class="texttype2">2015</span>
+            </div>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="wbtn">
+                <ul class="wbtn1">
+                    <li class="wbtn3">HTML</li>
+                    <li class="wbtn3">CSS</li>                    
+                    <li class="wbtn3">JavaScript</li>
+                </ul>              
+            </div>                 
+            <button class="btn">See project</button>                        
+          </div>
+        </div>   
+    </section>    
```

Comment:
- [ ] As stated in the project You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). For the project list there is some difference from the Figma design Just like Below.
- The Figma Design for the Project List .
![Capture](https://user-images.githubusercontent.com/93598189/228792269-252c73c4-bcb7-4411-bee3-59fd70ddb0e4.PNG)

- Your Project List Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/228792944-a5c0a589-ddc9-4a6a-b43a-20931102d70b.PNG)